### PR TITLE
Redesign of webview, simpler interface

### PR DESCRIPTION
**Pull request by axdra**
Redesign of web view, the new one have the same back button as the rest of the app, also now has a "share" button so you can open in external browser. #262 

## Old:
image
## New:
image

### CODE REVIEW

**Comment by whyer** [comment]
Very nice! 

Fix the linting errors and I will merge.
`yarn lint --fix` is your friend here.

I personally use https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint in VSCode to get all the errors and warnings in the editor.

File: packages/app/components/icon.component.tsx [hunk 1]
```diff
@@ -23,3 +23,4 @@ export const SMSIcon = uiIcon('message-square-outline')
 export const SettingsIcon = uiIcon('settings-outline')
 export const BookOpenIcon = uiIcon('book-open-outline')
 export const GlobeIcon = uiIcon('globe-outline')
+export const ShareIcon = uiIcon('share-outline')
```
Comment (whyer) [inline packages/app/components/icon.component.tsx:26]: The share icon is not commonly used on iOS. Consider change to browser-outline.
https://akveo.github.io/eva-icons/#/?searchKey=web&type=outline

File: packages/app/components/modalWebView.component.tsx [hunk 2]
```diff
@@ -2,10 +2,10 @@ import { useApi } from '@skolplattformen/api-hooks'
 import { Text } from '@ui-kitten/components'
 import URI from 'jsuri'
 import React, { useEffect, useState } from 'react'
-import { Modal, StyleSheet, TouchableOpacity, View } from 'react-native'
+import { Linking,Modal, StyleSheet, TouchableOpacity, View,SyntheticEvent } from 'react-native'
```
Comment (whyer) [inline packages/app/components/modalWebView.component.tsx:5]: There are a lot of styling errors in this file. Just use the linting tool (yarn lint --fix) to correct them.

File: packages/app/components/icon.component.tsx (hunk 1 above)
Comment (axdra) [reply packages/app/components/icon.component.tsx:26]: Isn't this more suitable in this case?
https://akveo.github.io/eva-icons/#/?searchKey=globe-outline&type=outline

Comment (axdra) [reply packages/app/components/icon.component.tsx:26]: Alternatively this 
https://akveo.github.io/eva-icons/#/?searchKey=external&type=outline

Comment (whyer) [reply packages/app/components/icon.component.tsx:26]: Yes! The last one external-link-outline I think is the best. The globe is used for localization on the login page.

**Pull request merged**